styx-api-service/src/test/java/com/spotify/styx/api/BackfillResourceTest.java

```diff
@@ -164,6 +174,8 @@
       .concurrency(1)
       .nextTrigger(Instant.parse("2017-01-01T05:00:00Z"))
       .schedule(Schedule.HOURS)
+      .created(currentTime)
+      .lastModified(currentTime)
```

Comment:
Please double check if it's really necessary to set the timestamps for all backfills here. Since the timestamps are optional fields, we should use them sparsely in tests.